src/components/instructor/ClasslistManagerComponents/AddUsersFromFile.vue

```diff
@@ -333,7 +333,7 @@ const addMergedUsers = async () => {
 		}
 		try {
 			await users.addCourseUser(new CourseUser(user));
-			const full_name = `${user.first_name as string} ${user.last_name as string}`;
+			const full_name = `${user.first_name as string } ${user.last_name as string}`;
```

Comment:
Now just remove the added space, and put this file back to the way it was in the `main` branch and this pull request is ready.

Reply:
Did you find this by eye? `npm run lint` or `quasar dev` doesn't catch these.  Or this something that falls under prettier instead of eslint?